programs/server/config.xml

```diff
@@ -1261,25 +1261,14 @@
 
     <!--
         OpenTelemetry log contains OpenTelemetry trace spans.
+
+        NOTE: this table does not use standard schema with event_date and event_time!
     -->
     <opentelemetry_span_log>
-        <!--
-            The default table creation code is insufficient, this <engine> spec
-            is a workaround. There is no 'event_time' for this log, but two times,
-            start and finish. It is sorted by finish time, to avoid inserting
-            data too far away in the past (probably we can sometimes insert a span
-            that is seconds earlier than the last span in the table, due to a race
-            between several spans inserted in parallel). This gives the spans a
-            global order that we can use to e.g. retry insertion into some external
-            system.
-        -->
-        <engine>
-            engine MergeTree
-            partition by toYYYYMM(finish_date)
-            order by (finish_date, finish_time_us, trace_id)
-        </engine>
         <database>system</database>
         <table>opentelemetry_span_log</table>
+        <partition_by>toYYYYMM(finish_date)</partition_by>
+        <order_by>finish_date, finish_time_us, trace_id</order_by>
```

Comment:
Actually `trace_id` looks useless as there will be no 8k rows with the same `finish_time_us`

Reply:
This makes total sense. It will be removed separately.

Reply:
https://github.com/ClickHouse/ClickHouse/pull/75907